common/monitoring/monitoring.go

```diff
@@ -71,13 +72,19 @@ func NewOpenTracing(apiURL string) (ShutdownFn, error) {
 	// Enable multi-span attributes
 	bsp := honeycomb.NewBaggageSpanProcessor()
 
+	hcApiKey := honeycombApiKey
+	if hcApiKey == "" {
+		hcApiKey = os.Getenv("HONEYCOMB_API_KEY")
+	}
```

Comment:
I have two questions on this: 
1) I couldn't find `honeycombApiKey` in this file; is it some golang magic or am I missing something?
2) at line 77, shouldn't we get this value from our appconfig module?

Reply:
1. The value of honeycombApiKey is dynamically added in the build time when a new release is generated
2. This value is used only for development, I will add a comment there.